tools/api-key-rotation/api_key_rotation_checker/main.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+
+#
+# Copyright 2021 Google LLC.
+# This software is provided as is, without warranty or representation for any use or purpose. # pylint: disable = line-too-long
+# Your use of it is subject to your agreement with Google.
+#
```

Comment:
Please use the Apache license copyright header.

Reply:
Updated